lib/prepArticles.js

```diff
@@ -2,17 +2,14 @@ const Mercury = require("@postlight/mercury-parser")
 const sentiment = require('./sentimentAnalysis.js')
 
 const {AVG_WORDS_PER_MINUTE} = require("../config.json")
-const scrappedContentType = 'html'
```

Comment:
Why tho

Reply:
Seemed very weird to have this const up here if its only used in place